### ISSUE TITLE
Error in HeaderValue (version 2.4.11 and 2.4.10) for PHP 5.3.26

### ISSUE THREAD

**Issue by BogMor** [zendframework/zend-mail#129]
In the versions zend-mail 2.4.10 and 2.4.11 I found a problem for the version PHP 5.3.26, in the file src/Header/HeaderValue.php.
The line 90 is not compatible with this version 5.3.26
`if ($lf !== 10 || ! in_array($sp, [9, 32], true)) {`
This line was in 2.4.9
`if ($lf !== 10 || $sp !== 32) {`

A solution is to replace `[9, 32]` with `array(9, 32)`

**Comment by glensc** [comment]
committed the fix:
https://github.com/glensc/zend-mail/commit/e00ac0101b964063c70e282575a5b1bc4022f227

but can't make PR as `zendframework/zend-mail` has no base branch. afaik they have some master monolithic tree for 2.4 releases.

i created PR to self, commits can be seen there:
https://github.com/glensc/zend-mail/pull/1

for maintainers, i guess they could add manually remote and pull from there.
or git cherry pick
or git am

this can be it:
```
wget https://github.com/glensc/zend-mail/pull/1.patch
git am 1.patch
```

**Comment by glensc** [comment]
btw, there are more 5.3 fixes i have in my [maintenance branch](https://github.com/glensc/zend-mail/tree/develop-2.4):

- fdd7e94e09f24b1c54a6e78a704b5a633ddb5541 - irrelevant for production as fix is in tests

can be applied again with appending `.patch` to url and `git am`

**Comment by glensc** [comment]
this is fixed in 2.4.13, @BogMor or @weierophinney close this
